lib/embed/viewer/common_viewer.rb

```diff
@@ -199,6 +199,14 @@ def file_url(title)
       end
 
       ##
+      # Creates a pretty date in a standardized sul way
+      # @param [String] date_string
+      # @return [String]
+      def sul_pretty_date(date_string)
+        Date.parse(date_string).strftime('%d-%b-%Y')
```

Comment:
I think adding `I18n` should be fine and I don't think we'll need to include the translation helper if we access it directly though `I18n` (e.g. `I18n.l(Date.parse(date_string), format: :something)`)